nodestream/databases/neo4j/query.py

```diff
@@ -12,41 +12,51 @@
 RETURN batches, committedOperations, failedOperations, errorMessages
 """
 
+NON_APOCH_COMMIT_QUERY = """
+UNWIND $param_sets AS param
+CALL apoc.cypher.doIt($batched_query, {params: param}) 
+YIELD value 
+RETURN value
+"""
 
 @dataclass(slots=True, frozen=True)
 class Query:
```

Comment:
`Query` represents a single query to be submitted to the database. I don't think we need to include `apoc_iterate` rules in here.